src/pivot-table/components/CellFactory.tsx

```diff
@@ -1,87 +1,52 @@
 import React from 'react';
-import { View, Text, StyleSheet } from 'react-native';
-import { Model } from '../../types/types';
-import { Cell, TYPE } from '../handle-data';
+import { ItemData } from '../../types/types';
+import { TYPE } from '../handle-data';
```

Comment:
TYPE looks like an enum. Not something that needs to be changed in this PR though. Also there might be a better name for it 😄

Reply:
Yeah it should be an enum. Not sure why I did do it like an object. Might have be that I ran into to some issue. Will take another look at it.